test/src/commands/test/test_test.dart

```diff
@@ -177,6 +178,27 @@ void main() {
       ).called(1);
     });
 
+    test('completes normally --test-randomize-ordering-seed random', () async {
+      when<dynamic>(
+        () => argResults['test-randomize-ordering-seed'],
+      ).thenReturn('random');
+      final result = await testCommand.run();
+      expect(result, equals(ExitCode.success.code));
+      verify(
+        () => flutterTest(
+          arguments: [
+            '--test-randomize-ordering-seed',
+            'random',
```

Comment:
Should we also allow the user to pass a seed?

Reply:
oops good catch! I had unstaged changed 😅

Reply:
Yup they can this is just a test where we're passing random. I can add another test for a seed value.

Reply:
Just added another test lmk what you think 👍